<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HardiskChecker</class>
 <widget class="QWidget" name="HardiskChecker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>141</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="HardiskCheckLabel">
     <property name="text">
      <string>Client IP :&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="HardiskCheckEdit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
